ExecDef/tools.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+from pathlib import Path
+
+from gooddata_pandas import GoodPandas
+from gooddata_sdk import Attribute, ExecutionDefinition, GoodDataSdk, ObjId, SimpleMetric
+
+
+def get_workspace_id() -> str:
+    sdk = GoodDataSdk.create_from_profile(profile="default", profiles_path=Path("./profiles.yaml"))
+    return sdk.catalog_workspace.list_workspaces()[0].workspace_id
+
+
+def answer_to_json(answer: str) -> dict:
+    """Transform answer to dict, no matter the format.
+
+    This is a bulldozer of a function.
+
+    Args:
+        answer (str): Answer from the OpenAI agent.
+
+    Returns:
+        dict: _description_
+    """
+    regex = r"\{(?:[^{}]|)*\}"
```

Comment:
Consider using re.compile to improve performance. It is a tip for a future. No need to do that now ;)

Reply:
Thanks! :)